src/utils/getUserInputByQuestion.js

```diff
@@ -1,15 +1,10 @@
-import readline from 'readline';
+import * as readline from 'node:readline/promises';
 
-export const getUserInputByQuestion = (question) => {
-  const rl = readline.createInterface({
-    input: process.stdin,
-    output: process.stdout,
-  });
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
 
-  return new Promise((resolve) => {
-    rl.question(question + '\n', (answer) => {
-      rl.close();
-      resolve(answer);
-    });
-  });
+export const getUserInputByQuestion = (question) => {
+  return rl.question(question + '\n');
 };
```

Comment:
```suggestion
export const getUserInputByQuestion = async (question) => {
  const answer = await rl.question(question + '\n');
  rl.close();
  return answer;
};
```

이렇게 close 해줄 수 있지 않을까요?

Reply:
가능할 것 같습니다!
제시해준 코드가 기존에 사용했던 코드(readline/promises 적용 전)라
뭔가 다른 방법은 없을까 고민을 했었는데 🤔
말씀해주신 내용으로 적용해보겠습니다!